data/org.v1993.NXDumpClient.appdata.xml.in

```diff
@@ -1,26 +1,26 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <component type="desktop">
   <name>NX Dump Client</name>
-	<id>org.v1993.NXDumpClient</id>
+    <id>org.v1993.NXDumpClient</id>
   <developer_name>v19930312</developer_name>
-  <summary>Client for dumping over USB with nxdumptool</summary>
-	<metadata_license>CC0-1.0</metadata_license>
-	<project_license>GPL-3.0-or-later</project_license>
+  <summary>Dump Switch games over USB</summary>
```

Comment:
"Dump Switch games over USB with nxdumptool" would be better, but fail the current 35 character limit.